pdc-client.spec

```diff
@@ -1,22 +1,35 @@
+# Enable Python 3 builds for Fedora
+# We could enable it for EPEL as well, but some dependencies (beanbag) are
+# missing there.
+# NOTE: do **NOT** change 'epel' to 'rhel' here, as this spec is also
+# used to do RHEL builds without EPEL
 %if 0%{?fedora}
-%global with_python3 1
+# If the definition isn't available for python3_pkgversion, define it
+%{?!python3_pkgversion:%global python3_pkgversion 3}
+%bcond_without  python3
 %else
+%bcond_with     python3
+%endif
+
+# Compatibility with RHEL. These macros have been added to EPEL but
+# not yet to RHEL proper.
```

Comment:
FYI. When we make pdc related codes upstream, we decide to not mention any redhat related info in comments/codes/tests.